h2o-r/h2o-package/R/kvstore.R

```diff
@@ -232,7 +233,7 @@ h2o.getModel <- function(model_id) {
 #' h2o.download_pojo(my_model, getwd())  # save to the current working directory
 #' }
 #' @export
-h2o.download_pojo <- function(model, path=NULL, getjar=NULL, get_jar=TRUE) {
+h2o.download_pojo <- function(model, path=NULL, getjar=NULL, get_jar=TRUE, jar_name="") {
```

Comment:
If you set jar_name="h2o-genmodel.jar" you don't have to later handle the 2 different options (line 273 doesn't have to be modified)